### ISSUE TITLE
Lab Report 2 Feedback

### ISSUE THREAD

**Issue by EdwinLauMack** [garrett-lam/cse15l-lab-reports#2]
**Bug/Input 1**
- [X] Commit screenshot for bug/input 1
- [X] Link to test file for bug/input 1
- [X] Symptom shown as command + output for bug/input 1
- [x] Description of relationship between bug, symptom, and input for bug/input 1
  - Please include a description between the bug, symptoms and input
 
**Bug/Input 2**
- [X] Commit screenshot for bug/input 2
- [X] Link to test file for bug/input 2
- [X] Symptom shown as command + output for bug/input 2
- [x] Description of relationship between bug, symptom, and input for bug/input 2
  - Please include a description between the bug, symptoms and input

**Bug/Input 3**
- [X] Commit screenshot for bug/input 3
- [X] Link to test file for bug/input 3
- [X] Symptom shown as command + output for bug/input 3
- [x] Description of relationship between bug, symptom, and input for bug/input 3
  - Please include a description between the bug, symptoms and input

Additional comments from your grader:
Good job! You are just missing a description of the relationship between the bug, the symptom and the input for each bug/input

Feel free to comment below if you have any questions about the rubric/feedback.

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by garrett-lam** [comment]
Hi, I'm not sure what you guys are looking for for "Description of relationship between bug, symptom, and input"

For example, if I redid input 1 would it be like this: Bug: program doesn't take into account when [ or ] or (or ) is not found in markdown file and Symptom: StringOutOfBoundException since you cannot add to index -1 for arraylists?

**Comment by EdwinLauMack** [comment]
Yes, that seems like a better description. In your selected changes you talk about how you fixed it and the output you got. Like you mentioned above, it would be ideal if you address what is the bug and what is the symptom of the bug/input file as well.
